src/applications/gqrx/remote_control_settings.ui

```diff
@@ -88,7 +87,6 @@
        </property>
        <item>
         <property name="text">
-         <string>::ffff:127.0.0.1</string>
```

Comment:
Same comment as above. The .ui files should only be edited in Qt Creator / designer.